bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/ScriptExecution.java

```diff
@@ -91,6 +91,10 @@ public static Timer createTimer(AbstractInstant instant, Procedure0 closure) {
             dataMap.put("procedure", closure);
             dataMap.put("timer", timer);
             JobDetail job = newJob(TimerExecutionJob.class).withIdentity(jobKey).usingJobData(dataMap).build();
+            if (TimerImpl.scheduler.checkExists(job.getKey())) {
+                TimerImpl.scheduler.deleteJob(job.getKey());
```

Comment:
ok, so it is unlikely to have a clash, but I would actually very much prefer more readable keys, especially in the light of https://github.com/eclipse/smarthome/issues/581. It is not directly related to this PR, so I will merge, but if you have a minute to think about better key names, I would appreciate it :-)